sq/src/commands/key.rs

```diff
@@ -378,7 +378,7 @@ fn _password(config: Config, m: &ArgMatches, key: Cert) -> Result<()> {
 }
 
 // Unlocks a cert with a passphrase
-fn _unlock(key: Cert) -> Result<Cert> {
+pub fn _unlock(key: Cert) -> Result<Cert> {
```

Comment:
Why do you need to expose this function? 

If you have good reason, please review the full function as it might assume things about context or the argument. Usually the `_` means the function is internal or ignored.